iothub/device/src/netstandard13/HsmAuthentication/Transport/BufferedReadStream.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Devices.Client.HsmAuthentication.Transport
+{
+    internal class BufferedReadStream : Stream
```

Comment:
Did you add this file?

Reply:
Exists in net std 2.0 but not in 1.3?

Reply:
yes, netstandard1.3 doesn't have BufferedStream

---
In reply to: [195528713](https://github.com/Azure/azure-iot-sdk-csharp/pull/512#discussion_r195528713) [](ancestors = 195528713)